src/scripts/generate-config.sh

```diff
@@ -53,9 +53,11 @@ function generateConfig() {
 
   touch "${PARAM_GENERATED_CONFIG_PATH}"
 
+  # shellcheck disable=SC2154,SC2016
   < "${PARAM_CONFIG_LIST_PATH}" \
-  awk 'NF {printf "\"%s\" ", $0}' \
-  | xargs -0 -I {} sh -c 'cue export {} --out yaml' \
+  awk '{$1=$1};1' \
```

Comment:
What does this line do?

Reply:
It removes leading and trailing whitespaces

Reply:
TIL! I had to look up how that works, and I understand it now. 

PS: [this](https://unix.stackexchange.com/a/568679) posts' explanation says to leave out the `;` -- I am not sure if it changes the action in any way.